.gitattributes

```diff
@@ -19,3 +19,8 @@
 /phpunit.xml.dist export-ignore
 /spec export-ignore
 /tests export-ignore
+
+###> squizlabs/php_codesniffer ###
```

Comment:
Those should be in `.gitignore` ?

Reply:
Yes.There is no need to comment on these files, as they will only lead to more confusion. In other open projects, they are also ignored =)